### ISSUE TITLE
Group Norm, Batch Norm, Instance Norm, which is better

### ISSUE THREAD

**Issue by GaoxiangLuo** [GaoxiangLuo/gaoxiangluo.github.io#36]
https://gaoxiangluo.github.io/2021/08/01/Group-Norm-Batch-Norm-Instance-Norm-which-is-better/ 

 Batch Norm (BN)   Figure 1: Multilayer Perceptron (MLP) without BN. Credit: Lou HD   Figure 2: Multilayer Perceptron (MLP) with BN. Credit: Lou HD PyTorch implementation of BN torch.nn.BatchNorm2d(num

**Comment by Ryan-Qiyu-Jiang** [comment]
Nice website and post! The cat is very cool :D, how do you get so many hearts in your coffee :o btw commenting on chrome seems to unselect the textbox

**Comment by GaoxiangLuo** [comment]
> @Ryan-Qiyu-Jiang
> Nice website and post! The cat is very cool :D, how do you get so many hearts in your coffee :o btw commenting on chrome seems to unselect the textbox

Thanks for reporting that! I didn't realize how buggy it is since you're literally the first person commenting on my blog. Btw, I like your website too! As for the latte arts of the coffee, I basically just youtubed and taught myself how to pour out a heart.
